### PR TITLE
Refactor testing utilities, misc

### PR DESCRIPTION
**Pull request by zpqrtbnk**

### CODE REVIEW

**Comment by zpqrtbnk** [comment]
verify

**Comment by zpqrtbnk** [comment]
Shhh... I've pushed code that builds in `Debug` mode but not in `Release` mode... fixing...

File: src/Hazelcast.Net.Testing/Logging/StringBuilderLoggerExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+{
+    public static class StringBuilderLoggerExtensions
+    {
+        public static ILoggingBuilder AddStringBuilder(this ILoggingBuilder builder, StringBuilder text, StringBuilderLoggerOptions options = null)
+        {
+            builder.AddConfiguration();
+
+            var descriptor = ServiceDescriptor.Singleton<ILoggerProvider>(new StringBuilderLoggerProvider(text, options));
+            builder.Services.TryAddEnumerable(descriptor);
+            //LoggerProviderOptions.RegisterProviderOptions<ConsoleLoggerOptions, ConsoleLoggerProvider>(builder.Services);
```
Comment (asimarslan) [inline src/Hazelcast.Net.Testing/Logging/StringBuilderLoggerExtensions.cs:31]: is this a leftover

**Review by asimarslan — APPROVED** [review]

File: src/Hazelcast.Net.Testing/Logging/StringBuilderLoggerExtensions.cs (hunk 1 above)
Comment (zpqrtbnk) [reply src/Hazelcast.Net.Testing/Logging/StringBuilderLoggerExtensions.cs:31]: ah no, I copied that code from the original .NET console provider, and left it as a reference.

**Pull request merged**